src/components/Navbar/Hamburger.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Link from 'next/link'
+import useTranslation from 'next-translate/useTranslation'
+
+const Hamburger = () => {
+  const paths = ['/products', '/about-us', '/sign-in', '/']
+  // TODO link to sign up page when done modal //
+  const HamburgerMenuItems = ['product', 'aboutUs', 'signIn', 'signUp']
+  const { t } = useTranslation('common')
+
+  const renderHamburgerIcon = () => (
+    <>
+      <span></span>
+      <span></span>
+      <span></span>
+    </>
+  )
+
+  const renderHamburgerMenu = () =>
+    paths.map((path, index) => (
+      <li className="hamburger__menu-list">
```

Comment:
```suggestion
      <li className="hamburger__menu-list" key={index}>
```
To eliminate this warning, please add key when using `map` function.

<img width="498" alt="Screen Shot 2564-04-06 at 23 13 30" src="https://user-images.githubusercontent.com/32285706/113743341-c6d36f80-972d-11eb-80a8-8b76b02a113b.png">